packages/app/components/Queue/QuestionForm.tsx

```diff
@@ -66,13 +72,35 @@ export default function QuestionForm({
   // on question type change, update the question type state
   const onCategoryChange = (e: RadioChangeEvent) => {
     setQuestionTypeInput(e.target.value);
+
+    const questionFromStorage = storageQuestion ?? {};
+
+    window.localStorage.setItem(
```

Comment:
can't you just `setStoredQuestion` here? I dont' know all of the details of the hook tho tbh